CherkasovaDA/lab_3/main.cpp

```diff
@@ -0,0 +1,260 @@
+#include <iostream>
+
+using namespace std;
+
+struct MyType
+{
+    int a;
+};
+
+class MyVector
+{
+    int size = 0;
+    int capasity = 0;
+    MyType* massiv = 0;
```

Comment:
There are feature called "type aliases" in C++
Instead of defining MyType as a separate struct you can define it like
`using MyType = int;`